core/tests/revert-test/tests/revert-and-restart.test.ts

```diff
@@ -150,7 +150,7 @@ describe('Block reverting test', function () {
     step('revert blocks', async () => {
         const executedProcess = await utils.exec(
             'cd $ZKSYNC_HOME && ' +
-                'RUST_LOG=off cargo run --bin block_reverter --release -- print-suggested-values --json'
+                'RUST_LOG=off cargo run --bin block_reverter --release -- print-suggested-values --json --operator-address 0xde03a0B5963f75f1C8485B355fF6D30f3093BDE7'
```

Comment:
Probably it's better to get it from config

Reply:
Does the address, not the key exist there ?

Reply:
for tests everything exists

Reply:
done, takes it from env now